src/custom_parser/query/model_parser.cpp

```diff
@@ -1,4 +1,4 @@
-#include "flockmtl/core/parser/query/model_parser.hpp"
+#include "flockmtl/custom_parser/query/model_parser.hpp"
```

Comment:
I don't think custom is needed.

Reply:
just to disinguish between our parser and the duckdb parser